asp_interactive_configuration/examples/racks/constraints.lp

```diff
@@ -1,183 +1,27 @@
-	active(new_object),
-    ooasp_isa(module,ID),
-    ooasp_isa_leaf(LEAFCLASS,ID),
-    LEAFCLASS != moduleV.
-
-
-ooasp_cv(module_requires_element,ID,"Modules of type {} must have at least one element",(LEAFCLASS,new_object)) :-
-	ooasp_arity(element_modules, 2, ID, ARITY, new_object),
-	active(new_object),
-    ooasp_isa(module,ID),
-    ooasp_isa_leaf(LEAFCLASS,ID),
-    LEAFCLASS != moduleV,
-    ARITY==0.
-
-ooasp_cv(frame_position_must_be_unique, new_object, "Frame position is equal to the one of frame {} ",(F,)):-
-    ooasp_associated(rack_frames,R,new_object),
-    ooasp_associated(rack_frames,R,F),
-    new_object!=F,
-    ooasp_attr_value(frame_position,new_object,P),
-    ooasp_attr_value(frame_position,F,P).
-
-ooasp_cv(frame_position_must_be_unique, F1, "Frame position is equal to the one of frame {} ",(F2,new_object)):-
-    ooasp_associated(rack_frames,new_object,F1),
-    ooasp_associated(rack_frames,new_object,F2),
-    F1!=F2,
-    ooasp_attr_value(frame_position,F1,P),
-    ooasp_attr_value(frame_position,F2,P).
-
-% frame positions for racksingle
-ooasp_cv(frame_must_be_within_range, new_object, "Frame position is outside of range {} ",(new_object,)):-
-    ooasp_isa(rackSingle,ID),
-    ooasp_associated(rack_frames,ID,new_object),
-    ooasp_attr_value(frame_position,F,P),
-    P<1.
+ooasp_potential_cv(moduleII_requires_moduleV).
 
+% Rack Single frames must have a position smaller than 4
 ooasp_cv(frame_must_be_within_range, new_object, "Frame position is outside of range {} ",(new_object,)):-
     ooasp_isa(rackSingle,ID),
     ooasp_associated(rack_frames,ID,new_object),
-    ooasp_attr_value(frame_position,F,P),
+    ooasp_attr_value(frame_position,new_object,P),
     P>4.
 
-ooasp_cv(frame_must_be_within_range, F, "Frame position is outside of range {} ",(F,new_object)):-
-    ooasp_isa(rackSingle,new_object),
-    ooasp_associated(rack_frames,new_object,F),
-    ooasp_attr_value(frame_position,F,P),
-    P<1.
-
 ooasp_cv(frame_must_be_within_range, F, "Frame position is outside of range {} ",(F,new_object)):-
     ooasp_isa(rackSingle,new_object),
     ooasp_associated(rack_frames,new_object,F),
     ooasp_attr_value(frame_position,F,P),
     P>4.
 
-% frame positions for rackdouble
-ooasp_cv(frame_must_be_within_range, new_object, "Frame position is outside of range {} ",(new_object,)):-
-    ooasp_isa(rackDouble,ID),
-    ooasp_associated(rack_frames,ID,new_object),
-    ooasp_attr_value(frame_position,F,P),
-    P<1.
-
-ooasp_cv(frame_must_be_within_range, new_object, "Frame position is outside of range {} ",(new_object,)):-
-    ooasp_isa(rackDouble,ID),
-    ooasp_associated(rack_frames,ID,new_object),
-    ooasp_attr_value(frame_position,F,P),
-    P>8.
-
-ooasp_cv(frame_must_be_within_range, F, "Frame position is outside of range {} ",(F,new_object)):-
-    ooasp_isa(rackDouble,new_object),
-    ooasp_associated(rack_frames,new_object,F),
-    ooasp_attr_value(frame_position,F,P),
-    P<1.
```

Comment:
Why has this and other `frame_must_be_within_range` constraints been removed?

Reply:
Redundant